diff -urN paragui-1.0.4.org/include/pgtimerobject.h paragui-1.0.4/include/pgtimerobject.h
--- paragui-1.0.4.org/include/pgtimerobject.h	2004-05-22 13:43:43.680927616 +0200
+++ paragui-1.0.4/include/pgtimerobject.h	2004-05-22 13:44:05.370630280 +0200
@@ -32,7 +32,7 @@
 #include "paragui.h"
 #include <map>
 
-typedef Uint32 PG_TimerID;
+typedef unsigned long PG_TimerID;
 
 /**
 	@author Alexander Pipelka
